app/uk/gov/hmrc/agentservicesaccount/views/pages/desi_details/check_updated_details.scala.html

```diff
@@ -18,45 +18,53 @@
 @import uk.gov.hmrc.agentservicesaccount.models.AgencyDetails
 @import uk.gov.hmrc.agentservicesaccount.views.html.main
 @import uk.gov.hmrc.govukfrontend.views.html.components._
-@import uk.gov.hmrc.agentservicesaccount.views.html.pages.desi_details.partials.contact_details_summary_list_partial
+@import uk.gov.hmrc.agentservicesaccount.views.html.pages.desi_details.partials._
 @import uk.gov.hmrc.agentservicesaccount.controllers.desiDetails
+@import uk.gov.hmrc.agentservicesaccount.models.desiDetails._
 
 @this(
-        main: main, h1: h1, p: p, ul: ul,
+        main: main, h1: h1, h2: h2, p: p, ul: ul,
         formWithCSRF: FormWithCSRF,
         button: GovukButton,
         inset_text:inset_text,
-        contact_details_summary_list_partial: contact_details_summary_list_partial
+        contact_details_cya_partial: contact_details_cya_partial,
+        other_services_cya: other_services_cya_partial,
+        your_details_cya: your_details_cya_partial
 )
 
 @(
         agencyDetails: AgencyDetails,
-        isAdmin: Boolean
+        isAdmin: Boolean,
+        otherServices: OtherServices,
+        submittedBy: YourDetails,
+        selectChanges: Set[String]
 )(implicit msgs: Messages, request: Request[_], appConfig: AppConfig)
 
 @main(
-    pageTitle = msgs("update-contact-details.review.title"),
+    pageTitle = msgs("update-contact-details.check-your-answers.title"),
     isAdmin = isAdmin,
     mimicBrowserBackLink = true, // as we can end up on this page from 4 different routes, the default browser 'back' works better than providing an explicit href
-    analyticsAdditionalJs = None
+    analyticsAdditionalJs = None,
+    showSecondaryNav = false
 ) {
 
-    @h1("update-contact-details.review.title")
-    @p("update-contact-details.review.p1")
+    @h1("update-contact-details.check-your-answers.title")
+    @p(key = "update-contact-details.check-your-answers.inset", classes = Some("govuk-inset-text"))
```

Comment:
Would it better to use the insetText helper here?

Reply:
Done